src/sys/socket/sockopt.rs

```diff
@@ -905,18 +905,31 @@ sockopt_impl!(
     libc::IP_PKTINFO,
     bool
 );
-#[cfg(any(linux_android, target_os = "freebsd", apple_targets, netbsdlike))]
+#[cfg(any(linux_android))]
 #[cfg(feature = "net")]
 sockopt_impl!(
     #[cfg_attr(docsrs, doc(cfg(feature = "net")))]
-    /// Set delivery of the `IPV6_PKTINFO` control message on incoming
+    /// Set delivery of the `IPV6_RECVPKTINFO` control message on incoming
```

Comment:
Emm, why this change? The old document seems to be correct, it asks the incoming packages to contain the `IPV6_PKTINFO` info

Reply:
@stevelauc This code is for the `IPV6_RECVPKTINFO` flag, The documentation incorrectly mentioned `IPV6_PKTINFO`

Reply:
It seems to be copied from the Linux manual: https://man7.org/linux/man-pages/man7/ipv6.7.html

> IPV6_RECVPKTINFO (since Linux 2.6.14)
              Set delivery of the IPV6_PKTINFO control message on
              incoming datagrams.  Such control messages contain a
              struct in6_pktinfo, as per RFC 3542.  Allowed only for
              SOCK_DGRAM or SOCK_RAW sockets.  Argument is a pointer to
              a boolean value in an integer.

Reply:
Ah my bad, Reverting it now